doc/rtd/development/code_review.rst

```diff
@@ -1,236 +0,0 @@
-.. _code_review_process:
```

Comment:
This page had a couple of useful tidbits and a lot of words to essentially describe the state machine of open source contributions. Extract the interesting bits and retire the page.